src/traces/table/plot.js

```diff
@@ -0,0 +1,82 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var table = require('./render');
+
+module.exports = function plot(gd, cdTable) {
```

Comment:
the latest push fixes it - it wasn't a simple file concat, the main dataflow has been plotlified, eg. referencing `calcdata` etc. and phasing out `model` and `viewModel`, hopefully you had something like this in mind